### PR TITLE
Require 0.93 validation set accuracy

### PR DESCRIPTION
**Pull request by andrewpaster**

### CODE REVIEW

File: Traffic_Sign_Classifier.ipynb [hunk 1]
```diff
@@ -157,16 +157,18 @@
     "\n",
     "Design and implement a deep learning model that learns to recognize traffic signs. Train and test your model on the [German Traffic Sign Dataset](http://benchmark.ini.rub.de/?section=gtsrb&subsection=dataset).\n",
     "\n",
+    "The LeNet-5 implementation shown in the [classroom](https://classroom.udacity.com/nanodegrees/nd013/parts/fbf77062-5703-404e-b60c-95b78b2f3f9e/modules/6df7ae49-c61c-4bb2-a23e-6527e69209ec/lessons/601ae704-1035-4287-8b11-e2c2716217ad/concepts/d4aca031-508f-4e0b-b493-e7b706120f81) at the end of the CNN lesson is a solid starting point. You'll have to change the number of classes and possibly the preprocessing, but aside from that it's plug and play! \n",
+    "\n",
+    "With the LeNet-5 solution from the lecture, you should expect a validaiton set accuracy of about 0.89. To meet specificaitons, the validation set accuracy will need to be at least 0.93. It is possible to get an even higher accuracy, but 0.93 is the minimum. \n",
```
Comment (domluna) [inline Traffic_Sign_Classifier.ipynb]: typo validaiton -> validation

Comment (domluna) [reply Traffic_Sign_Classifier.ipynb]: specificaitons -> specifications

Comment (domluna) [reply Traffic_Sign_Classifier.ipynb]: "It is possible to get an even higher accuracy, but 0.93 is the minimum."

"It is possible to get an even higher accuracy, but 0.93 is the minimum for a successful project submission."

Might not be super clear what the minimum is for.

**Review by andrewpaster — COMMENTED** [review]
Changed the misspellings and added clarifications

**Pull request merged**